framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -102,29 +114,28 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 
 	public async load(): Promise<void> {
 		this._chainConnectorPluginDB = await getDBInstance(this.dataPath);
-		this._sidechainChainConnectorStore = new ChainConnectorStore(
-			this._chainConnectorPluginDB,
-			DB_KEY_SIDECHAIN,
-		);
-		this.endpoint.load(this._sidechainChainConnectorStore);
+		this._chainConnectorStore = new ChainConnectorStore(this._chainConnectorPluginDB);
+		this.endpoint.load(this._chainConnectorStore);
 
-		this._mainchainAPIClient = await apiClient.createIPCClient(this.config.mainchainIPCPath);
-		if (this.config.sidechainIPCPath) {
-			this._sidechainAPIClient = await apiClient.createIPCClient(this.config.sidechainIPCPath);
+		this._receivingChainClient = await apiClient.createIPCClient(this.config.receivingChainIPCPath);
+		if (this.config.sendingChainIPCPath) {
+			this._sendingChainClient = await apiClient.createIPCClient(this.config.sendingChainIPCPath);
 		} else {
-			this._sidechainAPIClient = this.apiClient;
+			this._sendingChainClient = this.apiClient;
 		}
```

Comment:
2 IPC paths in case if someone wants to run the plugin stand alone but maybe we can simplify it?
and I think WS is a good idea as there is a good chance that the other chain node is outside local machine

Reply:
> 2 IPC paths in case if someone wants to run the plugin stand alone but maybe we can simplify it?

so far, there is no way to run the plugin in standalone mode

Reply:
I have removed `sendingChainIPCPath` and added `receivingChainWsURL`